packages/office-ui-fabric-react/src/components/pickers/Suggestions/__snapshots__/Suggestions.test.tsx.snap

```diff
@@ -801,7 +801,7 @@ exports[`Suggestions renders a list properly 1`] = `
               &:active .ms-Button-icon {
                 color: #004578;
               }
-          data-is-focusable={true}
+          data-is-focusable={false}
           disabled={undefined}
```

Comment:
This seems wrong, if disabled is undefined, shouldn't the data be focusable?

Reply:
This was a generated change from doing "npm run update-snapshots" so I didn't actually make this change... I'm very new to snapshot tests so not sure if this is correct however I do know that data-is-focusable doesn't actually control much behaviour & instead is just a metadata flag that just instructs FocusZone on how to treat the element. If you know more about this, I'd love to get a better understanding of the consequences of changing this flag's value to false in these snapshot tests.

Reply:
data-is-focusable ends up controlling a lot when items are within focus zones, if it's set to false, then the focus-zone will never give them focus, even in this case, when it probably should get focus.